aws/build.gradle

```diff
@@ -162,3 +164,33 @@ def getCurrentGitBranch() {
     }
     return gitBranch
 }
+
+task installSpreadPythonDependency(type: Exec) {
+    workingDir 'src/main/python/spreadsheet_lambda/package'
+    executable 'python3'
+    args '-m', 'pip', 'install', '-r', 'requirements.txt', '-t', '.'
+}
+
+task packageSpreadSheetFunction(dependsOn: [installSpreadPythonDependency], type: Zip) {
+    from fileTree('src/main/python/spreadsheet_lambda/package')
+    from ('src/main/python/spreadsheet_lambda/lambda_function.py')
+    from ('src/main/python/spreadsheet_lambda/credentials.json')
```

Comment:
How is this filled? I don't see this file in the PR? Same for the next two.

Reply:
I've removed `credentials.json`